packages/headless/src/features/search-parameters/search-parameter-actions.ts

```diff
@@ -1,12 +1,22 @@
 import {createAction} from '@reduxjs/toolkit';
-import {
-  EnableQuerySyntaxParam,
-  QueryParam,
-} from '../../api/search/search-api-params';
+import {SearchRequest} from '../../api/search/search/search-request';
 import {validatePayloadSchema} from '../../utils/validate-payload';
 import {searchParametersDefinition} from './search-parameter-schema';
 
-export type SearchParameters = QueryParam & EnableQuerySyntaxParam;
+export type SearchParameters = Omit<
```

Comment:
I switched the `SearchParameters` type to derive from `SearchRequest`. Adding a new param to the request (e.g. `debug`) will alert the person to consider whether it should be supported in the url. If not, then it only needs to be added to the list below.